src/core/Button/style.ts

```diff
@@ -30,13 +47,26 @@ const ButtonBase = styled(Button, {
 
     const padding = variant === "outlined" ? outlinedPadding : containedPadding;
 
+    const border = borders && borders[colorProp];
+    const color = colors && colors[colorProp];
+
+    if (!border) {
+      throw new Error(`Button border not found: ${colorProp}`);
+    }
+    if (!color) {
+      throw new Error(`Button color not found: ${colorProp}`);
+    }
+
     return `
+      background-color: ${color[400]};
+      border: ${border[400]};
+      color: white;
       padding: ${padding};
       min-width: 120px;
       height: 34px;
       &:hover, &:focus {
         color: white;
-        background-color: ${colors?.primary[500]};
+        background-color: ${color[600]};
```

Comment:
There is no `500` defined for error color, so I just changed this to use 600 for any color.  The other option would be to add a `500` color for error.  Note that this makes the hover color the same as active below (which was previously `600`).

Reply:
Ahh thanks for pointing this out! 

@clarsen-czi @hthomas-czi Should we define error 500 value in this case? 

Thanks all!

Reply:
I think the ideal behavior would be to use `500` if that value exists, otherwise use `600`.

Reply:
I agree with Harley—I think we'd ideally use `error500` which doesn't exist. I've just reached out to the designers on the design system team Slack to make sure they're ok with us adding it in. I'll follow up once I hear back

Reply:
thanks for chiming in! adding `error500` sounds like a good path.  Then we can consistently have separate hover and active colors for all button types.  I'm happy to add in the `error500` color once we've got it!

Reply:
TODO: Add `error500` color (#C61128), and use 500 color for hover.

https://czi-sci.slack.com/archives/C029M3YCFFD/p1643844470244839?thread_ts=1643674010.289539&cid=C029M3YCFFD